repository/src/main/resources/db/changelog/db.changelog-rdm-10041.xml

```diff
@@ -0,0 +1,116 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-10041" author="rebecca.baker@hmcts.net">
+        <!-- FixedList40 -->
+        <insert tableName="field_type">
+            <column name="created_at" value="${now}" />
+            <column name="reference" value="FixedList40" />
+            <column name="version" value="1" />
+            <column name="maximum" value="40" />
+        </insert>
```

Comment:
Although this is in the requirements, creating a new base type seems like adding more complexity, if it is simply a known base type with constraints.. Wonder whether it'd be better making it simply use `FixedList` and then add a new field ID validator (in progress PR - https://github.com/hmcts/ccd-data-store-api/pull/1194).

Reply:
@danlysiak Spoke with Jamal agreed we could use FixedList and removed the 40 restriction. Reverted this change out